modules/context/context.go

```diff
@@ -687,6 +677,8 @@ func getCsrfOpts() CsrfOptions {
 	}
 }
 
+const CookieNameFlash = "gitea_flash"
```

Comment:
can we move this at the top of the file?

Reply:
I personally prefer constants to be placed at the beginning of the file for better discoverability

Reply:
Done in 4e5d16f1d1ba953fcb87d3aac5a3c1f1f4031f5c, but TBH I don't think there is any difference. In this file .... you can see:

![image](https://user-images.githubusercontent.com/2114189/231818055-2d9ae41b-74f2-4ddf-9fbc-8b997e0721ae.png)

![image](https://user-images.githubusercontent.com/2114189/231818098-fe626fe7-f28d-45ec-85c3-ab3a1b752d9c.png)

![image](https://user-images.githubusercontent.com/2114189/231818126-5442305f-c8e8-4ce6-8d4b-1410d76add90.png)